rbx_binary/src/deserializer.rs

```diff
@@ -474,7 +482,35 @@ impl<R: Read> BinaryDeserializer<R> {
             }
             Type::Ray => {}
             Type::Faces => {}
-            Type::Axes => {}
+            Type::Axes => match canonical_type {
+                VariantType::Axes => {
+                    for referent in &type_info.referents {
+                        let instance = self.instances_by_ref.get_mut(referent).unwrap();
+                        let value = chunk.read_u8()?;
+                        let rbx_value = Variant::Axes(if let Some(axes) = Axes::from_bits(value) {
+                            axes
+                        } else {
+                            return Err(InnerError::InvalidPropData {
+                                type_name: type_info.type_name.clone(),
+                                prop_name,
+                                valid_value: "less than 7",
+                                actual_value: format!("{}", value),
+                            });
+                        });
```

Comment:
Can you write this block using [`Option::ok_or_else`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.ok_or_else) and the question mark operator instead? It'll turn out a little nicer!